modin/distributed/dataframe/pandas/partitions.py

```diff
@@ -89,7 +89,9 @@ def _unwrap_partitions(oid):
         ]
 
 
-def from_partitions(partitions, axis):
+def from_partitions(
```

Comment:
Let's add a note on additional arguments. Something like that: you can pass `index`, `columns`, `row_lengths` and `column_widths` to this function in order to avoid triggering extra computations to compute the metadata.

Reply:
Notes were added in docstring